dask_kubernetes/core.py

```diff
@@ -508,6 +508,35 @@ def f(to_close):
     def scale_up(self, n, **kwargs):
         self.scheduler.loop.add_callback(self._scale_up, n, **kwargs)
 
+    async def _pod_status(self, pods):
```

Comment:
```suggestion
   def _pod_status(self, pods):
```

This doesn't appear to need to be asynchronous

Reply:
Actually, this function only appears to be used once, and is relatively simple.  I recommend that we just inline this logic into the list comprehension where it is used.